libraries/src/MVC/Controller/ApiController.php

```diff
@@ -454,6 +454,9 @@ protected function save($recordKey = null)
 		// TODO: Not the cleanest thing ever but it works...
 		Form::addFormPath(JPATH_COMPONENT_ADMINISTRATOR . '/forms');
 
+        // Needs to be set because com_fields needs the data in jform to determine the assigned catid
+        $this->input->set('jform', $data);
```

Comment:
```suggestion
		// Needs to be set because com_fields needs the data in jform to determine the assigned catid
		$this->input->set('jform', $data);
```
Make Drone PHPCS check happy.